### PR TITLE
feat:guest look up table implement with global search bar

### PR DESCRIPTION
**Pull request by ekram2d**
i crated a table show this   Name | Phone | Address | Hotel | ID No | ID Type | Type information's  in a table with global search bar

### CODE REVIEW

**Comment by ekram2d** [comment]
i solved deploy issue now its work

**Comment by ahmad-ml** [comment]
Don't show the hotel in the table

**Comment by ekram2d** [comment]
ok sir

**Comment by ekram2d** [comment]
Now i also added update feature  all the fields of Guest Lookup table

File: src/pages/Transactions/Transactions.tsx [hunk 1]
```diff
@@ -140,7 +140,7 @@ const Transactions = () => {
     : transactions;
 
   // Filter transactions based on search text
-  const filteredTransactions = combinedTransactions.filter((transaction) => {
+  const filteredTransactions = combinedTransactions.filter((transaction:any) => {
```
Comment (ahmad-ml) [inline src/pages/Transactions/Transactions.tsx:143]: @ekram2d  why any type?

File: src/components/BookingSummary.tsx [hunk 2]
```diff
@@ -192,7 +192,7 @@ const BookingSummary = ({
         message.success("Transaction created successfully!");
         form.resetFields();
         setIsModalOpen(false);
-        setTransactionInfo(res?.data?.createTransaction);
+        setTransactionInfo(res?.data?.createTransaction as any);
```
Comment (ahmad-ml) [inline src/components/BookingSummary.tsx:195]: @ekram2d  any type is unexpected

Comment (ekram2d) [reply src/components/BookingSummary.tsx:195]: Sorry Sir i forget to change my any type after finishing works..

**Pull request closed without merging**